src/app/[locale]/seller/registration/page.tsx

```diff
@@ -388,6 +401,34 @@ const SellerRegistrationForm = () => {
           <ToggleCollapse
             header={t('SCREEN.BUY_FROM_SELLER.SELLER_CONTACT_DETAILS_LABEL')}
             open={false}>
+                <div className="mb-4">
```

Comment:
Can we just copy over what we had in the User Preferences sidenav?  So something like this ->
```
<Input
              label={t('SCREEN.SELLER_REGISTRATION.EMAIL_LABEL')}
              type="email"
              name="email"
              style={{
                textAlign: 'center'
              }}
              value={formData.email? formData.email: ""}
              onChange={handleChange}
            />
```

Reply:
Correct, this was an oversight, thanks for catching it!